formstack-baton-requests/src/main/scala/com/gu/identity/formstackbatonrequests/services/FormstackService.scala

```diff
@@ -74,30 +75,38 @@ object FormstackService extends FormstackRequestService with LazyLogging {
     else
       decode[FormSubmissions](response.body)
   }
-
+  
+  val skippableErrorMessages = List(
+    "An error occurred while decrypting the submissions",
```

Comment:
@ob6160  I don't remember why this was added ( I think it was a previous issue) but are we sure we want to skip this error ?

Reply:
No, we're not. It's not clear in what situations this error would arise.

Reply:
For this PR though we could keep it the same, to avoid changing current behaviour.

Reply:
I think the current behaviour current behaviour might be wrong..
It catches the "decoding" error but then it still returns a `Left` with a custom error that doesn't get any special treatment later so it still fails the whole thing, just with a different error:
https://github.com/guardian/identity-processes/pull/90/files

Reply:
I think it was modelled after the behaviour when dynamodb is updated with new form data but I think that case it works because:

1. The function that returns the left only makes a single api call so it's fine to return a `Left`. In this one we are in a loop so we want to continue with the next submission and  return a `Right` with a list of all the successful submissions unless there's an unskippable error in which case we would want to return a `Left`
2. The function that calls that has a special case to deal with the custom `FormstackDecryptionError` and not return an error. See [Here](https://github.com/guardian/identity-processes/blob/e92e3ae5becc2419769d35159cb3924c8a23afaa/formstack-baton-requests/src/main/scala/com/gu/identity/formstackbatonrequests/services/DynamoUpdateService.scala#L39)

Reply:
In short I think we should probably not skip this error. What do you think @ob6160 ?

Reply:
i think we can remove it from the list because on line 73 we have the check from before we came to the codebase: https://github.com/guardian/identity-processes/pull/92/files#diff-2e97f551ddedcd9a50db834124cfb109ad0dc4f748e72de9f9b84623d8625f0bR73